cpu/Makefile.include.cortexm_common

```diff
@@ -16,7 +16,9 @@ export CFLAGS += $(CFLAGS_CPU) $(CFLAGS_LINK) $(CFLAGS_DBG) $(CFLAGS_OPT)
 
 export ASFLAGS += $(CFLAGS_CPU) $(CFLAGS_DBG)
 export LINKFLAGS += -L$(RIOTCPU)/$(CPU)/ldscripts -L$(RIOTCPU)/cortexm_common/ldscripts
-export LINKFLAGS += -T$(RIOTCPU)/$(CPU)/ldscripts/$(CPU_MODEL).ld -Wl,--fatal-warnings
+export LINKER_SCRIPT ?= $(RIOTCPU)/$(CPU)/ldscripts/$(CPU_MODEL).ld
```

Comment:
Is it really needed to have the whole path of the .ld file? I mean, why don't we just have this (default value) as:

    export LINKER_SCRIPT ?= $(CPU_MODEL).ld

In the line above it, the LINKFLAGS is set with two paths to search for that .ld file